boilerplate/registry/index.html

```diff
@@ -0,0 +1,237 @@
+      <p>The intent is that this boilerplate can form the basis of [=w3c registries=] for
+        the W3C <dfn data-abbr="TTWG"><a href="https://www.w3.org/AudioVideo/TT/">Timed Text Working Group</a></dfn>.</p>
+      <p>See discussion <a href="https://github.com/w3c/ttwg/discussions/241">w3c/ttwg#241</a> for
+        details of the assumptions made to generate this document.</p>
+    </section>
+    <section id="sotd">
+      <p>This document is intended to be the basis of other Registry documents
+        published by TTWG and is not intended to be published as a Registry itself.
+        The following SOTD text is auto-generated by the Respec tool to reflect
+        what an actual Registry Definition document would contain.</p>
+    </section>
+    <section class="informative">
+      <h2>Introduction</h2>
+      <p>Some informative introductory text about this Registry.</p>
+    </section>
+    <section id="registry-definition">
+      <h2>Registry Definition</h2>
+      <p class="ednote">Insert sections on custodianship and change process.</p>
+      <section id="custodianship">
+        <h3>Custodianship</h3>
+        <p>The <dfn data-cite="w3c-process#custodian">custodian</dfn> of this Registry is the <a>TTWG</a>. 
+          If the <a>TTWG</a> is unable to fulfil the role of <a>custodian</a>,
+          for example if it has been <a data-cite="w3c-process#GeneralTermination">closed</a>,
+          the <a>custodian</a> in lieu is the <a data-cite="w3c-process#Team">W3C Team</a>.
+        </p>
+        <p class="issue" data-number="244"></p>
+      </section>
+      <section id="change-process">
+        <h3>Change Process</h3>
+        <section id="requesting-a-change">
+          <h4>Requesting a change</h4>
+          <p>Changes to this [=W3C Registry=] MUST be requested (the <dfn>change request</dfn>) using any one of the following options:</p>
+          <ul>
+            <li>Open an issue on the Registry Table's version control system;</li>
+            <li>Send an email to the <a>TTWG</a>'s public email reflector
+              <a href="mailto:public-tt@w3.org">public-tt@w3.org</a></li>
+          </ul>
+          <p>The <a>change request</a> MUST include enough information for the <a>custodian</a>
+            to be able to identify all of:</p>
+          <ul>
+            <li>The Registry Table;</li>
+            <li>The Entries being proposed for addition;</li>
+            <li>The Provisional Entries for which changes are being proposed;</li>
+            <li>The Entries being proposed for a Status change;</li>
+            <li>Details of the requested changes for each Entry;</li>
+            <li>The motivation for making the requested changes;</li>
+            <li>Any other supporting information, for example real world usage statistics;</li>
+          </ul>
+          <p>The proposer of the change MAY open a pull request (or equivalent) on
```

Comment:
Do we need some text that custodian shall open a pull request when it is not made (e.g. proposed via email)?
A note following this states that a PR is required before processing any review.

Reply:
That's already present in §2.2.2.1 first bullet:

>If the change proposer did not open a pull request on the [registry table](https://www.w3.org/Consortium/Process/#registry-table)'s version control system, then assessment is paused until a TTWG member has opened such a pull request, which MUST represent the requested changes and MUST be linked to a related issue.

Reply:
Ah, it's written in 2.2.2.1, sorry.